domains/GH9.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "GH9yt.is-a.dev",
+  "repo": "https://gh9youtube.github.io/GH9.github.io/",
+  "owner": {
+    "username": "GH9",
+    "email": "shyshyleprodu62@gmail.com"
+  },
+  "record": {
+    "CNAME": "GH9.github.io/"
```

```suggestion
    "CNAME": "GH9.github.io"
```
`CNAME` can't contain `/`